src/pip/_internal/cli/progress_bars.py

```diff
@@ -158,8 +150,7 @@ def iter(self, it):  # type: ignore
 
 
 class WindowsMixin:
-    def __init__(self, *args, **kwargs):
-        # type: (List[Any], Dict[Any, Any]) -> None
+    def __init__(self, *args: List[Any], **kwargs: Dict[Any, Any]) -> None:
```

Comment:
… and here (I should have bundled these comments together)

Reply:
Don't worry. I'll solve the last 2 changes in one commit.

Reply:
Done.